src/scene/vcFlythrough.cpp

```diff
@@ -389,3 +387,40 @@ void vcFlythrough::UpdateCameraPosition(vcState *pProgramState)
     }
   }
 }
+
+void vcFlythrough::LoadFlightPoints(vcState *pProgramState)
+{
+  udDouble3 *pFPPositions;
+  int numFrames;
+  vcProject_FetchNodeGeometryAsCartesian(&pProgramState->activeProject, m_pNode, pProgramState->geozone, &pFPPositions, &numFrames);
+
+  m_flightPoints.Clear();
+  m_flightPoints.GrowBack(numFrames);
```

Comment:
This is going to grow the chunked array by the number of frames every time the points are loaded. The parameter is `numberOfNewElements`, not `newCapacity`.